mmv1/products/spanner/Database.yaml

```diff
@@ -158,3 +159,11 @@ properties:
       - :GOOGLE_STANDARD_SQL
       - :POSTGRESQL
     default_from_api: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'enableDropProtection'
```

Comment:
You may see test failed for update due to that the resource is marked as immutable and therefore this field also has `forcenew:true` (see https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-8011-old..auto-pr-8011#diff-d3c76e2ae0007c894cbb4c0981c5d2257f519868321c7381a93a864fa6ff875eR184), which means it'll delete + create the resource based on the updated config when we try to update the field. However, since we have `enable_drop_protection = true` set when creating the resource in your test, you may see errors like the resource can't be deleted. We could add update_url + update_verb + update_mask(?) here to mark the field as mutable I think.

Also, another point is that we probably want to update this field first before we update other field? 
For example, if the user change `enable_drop_protection` from `true` to `false` along with some changes to other immutable fields in their config, we need to send this field first to disable to the protection and then delete and recreate the resource, otherwise, it wouldn't allow the deletion.
More importantly, if they change `enable_drop_protection` from `false` to `true` + changes to immutable fields, we definitely want to apply it first to prevent any data loss, otherwise, it will delete and create a new resource and then enable the drop_protection to the new resource.